app/webhook.go

```diff
@@ -150,10 +150,10 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 	}
 }
 
-func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType, secret string) (*model.OutgoingWebhookResponse, error) {
+func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType, secretToken string) (*model.OutgoingWebhookResponse, error) {
 	bodyBytes, err := ioutil.ReadAll(body)
 	if err != nil {
-		return nil, errors.New("OutgoingWehookRequest payload failed to read.")
+		return nil, fmt.Errorf("failed to read outgoing webhook request payload: %w", err)
```

Comment:
@jespino, can you provide guidance here? I know we recently discussed the use of `fmt.Errorf` vs. our traditional errors wrapping, and I vaguely recall one shortcoming re: missing stack traces when an error occurs. Should we start using `fmt.Errorf` or stick with the old pattern?

Reply:
Unlike the model.NewError stuff, one shortcoming with fmt.Errorf can be the i18n, isn't it?